admin/projects.go

```diff
@@ -247,12 +256,37 @@ func (s *Service) UpdateProjectVariables(ctx context.Context, proj *database.Pro
 	return s.DB.UpdateProjectVariables(ctx, proj.ID, variables)
 }
 
+// UpdateProjectAnnotations pushes the updated annotations to deployments.
+// NOTE : this does not trigger reconcile.
+func (s *Service) UpdateProjectAnnotations(ctx context.Context, proj *database.Project, annotations map[string]string) error {
```

Comment:
1. I was confused about what "project annotations are – maybe `UpdateDeploymentAnnotations` would be a better name?
2. Instead of accepting a free-form `map` here, I think it should take an org and project object, and then the function should internally construct the payload we expect